src/Raven.Server/Documents/Queries/QueryRunner.cs

```diff
@@ -58,6 +58,8 @@ public override async Task<DocumentQueryResult> ExecuteQuery(IndexQueryServerSid
             {
                 try
                 {
+                    documentsContext.CloseTransaction();
```

Comment:
Do we need to apply this to other retried queries? Check where we are using `CreateRetriesFailedException`

Reply:
done